.github/workflows/ansible-test.yml

```diff
@@ -146,6 +146,12 @@ jobs:
         exclude:
           - ansible: stable-2.9
             python: 3.9
+          - ansible: devel
+            python:
+              - 2.7
+              - 3.5
+              - 3.6
+              - 3.7
```

Comment:
I don't think you can use a list here. It seems like your checks did run devel with 2.7 for example.

Instead you need to add a separate exclude entry for every combination, unfortunately.

BTW, I will also recommend you take this time to change all your python version entries to strings by quoting them. These work fine until you try to add in `3.10` and you get errors about python 3.1.

Reply:
good spot, will do, thank you!